releng/jenkins/bot-updates/Jenkinsfile

```diff
@@ -289,30 +267,30 @@ pipeline {
           // set pull request title message
           env.PR_TITLE="[releng] Update Orbit URL to ${params.UPDATE_VALUE}"
 
-          script {
-            currentBuild.displayName = "#${env.BUILD_NUMBER} (Orbit URL)";
-          }
+          currentBuild.displayName = "#${env.BUILD_NUMBER} (Orbit URL)";
 
           REPOSITORY_NAMES.split(',').each {
             dir (it) {
               sh """
               for f in \$(find . -name *.target -type f); 
               do
-                sed -i -e 's|location=".*/orbit/.*"|location="${params.UPDATE_VALUE}"|' \$f
+                sed -i -e 's|https://.*/orbit/[^"]*"|${params.UPDATE_VALUE}|' \$f
               done
               """
             } // dir
           } // for each repository
-          REPOSITORY_NAMES.split(',').each {
-            dir (it) {
-              sh """
-                git diff-index --quiet HEAD || git commit --signoff -a -m "${env.PR_TITLE}"
-              """
-            } // dir
-          } // for each repository
+          if (fileExists('xtext-core')) { // for the case that we test without xtext-core repo; would fail otherwise
```

Comment:
should we also patch oomph here?

Reply:
Sure, when we are already on it. Done.
https://ci.eclipse.org/xtext/job/releng/job/bot-updates/84/console